rv_v

```diff
@@ -411,6 +427,13 @@ vwmaccu.vv     31..26=0x3c vm vs2 vs1 14..12=0x2 vd 6..0=0x57
 vwmacc.vv      31..26=0x3d vm vs2 vs1 14..12=0x2 vd 6..0=0x57
 vwmaccsu.vv    31..26=0x3f vm vs2 vs1 14..12=0x2 vd 6..0=0x57
 
+#Pseudo-Instructions for Vector Integer Extension Instructions
+
+$pseudo_op rv_v::vmxor.mm   vmclr.m   31..26=0x1b 25=1  vs2=vd  vs1=vd  14..12=0x2 vd 6..0=0x57
```

Comment:
More of a question than a comment: is there a reason for using vs2=vd rather than vs2=vs 1 here?  By transitivity, they're equivalent, just curious if there should be a stylistic default and, if so, why.

Reply:
This is to ensure that the parsing assumes them as the same value. This is extremely useful when generating, per example, the yaml_dump since we can know what differs from the original instruction to the pseudo-instruction.

This can be parsed to ouput, per example, the following:   https://github.com/riscv-software-src/riscv-unified-db/blob/00213d2b2703240dc4fbe90dcdd8749e5b5f7e35/arch/inst/B/add.uw.yaml#L25

Reply:
I think you missed my point.  There are two equivalent ways of expressing this: we can say vs2=vd and vs1=vd, or we can say vs1=vd and vs2=vs1.  The same information would be conveyed to parsing tools either way.  I am just wondering why we are anchoring on vd.

Reply:
I definitely missed it, but I believe the best way to go for it is to say that "original_field = other_field", since parsing wise this is more helpful